modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -213,10 +238,11 @@
 	icon = 'modular_skyrat/modules/customization/icons/~donator/obj/clothing/suits.dmi'
 	worn_icon = 'modular_skyrat/modules/customization/icons/~donator/mob/clothing/suit.dmi'
 	icon_state = "furcoat"
-	inhand_icon_state = "furcoat"
+	inhand_icon_state = "hostrench"
 	blood_overlay_type = "coat"
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS
 	cold_protection = CHEST|GROIN|LEGS|ARMS
+	mutant_variants = NONE
 
 //Donation reward for Thedragmeme
 /obj/item/clothing/under/syndicate/tacticool/black
```

Comment:
Same issue as above, hides the additional bodyoverlays (markings and such), and does not show the SUIT placeholder, but sprite does not show up either.

Reply:
to be clear, this is about `/obj/item/clothing/under/syndicate/tacticool/black` - expected github to show that line on the top and not the bottom on the comment.